main.js

```diff
@@ -466,7 +466,10 @@ async function cacheIt(text, language) {
         // find out if say.mp3 must be generated
         if (!speech2device || !speech2device.sayItIsPlayFile(text)) {
             const output = await getArrayMemberByValue(sayitOptions, 'value', adapter.config.type);
-            isGenerate = output.options.mp3Required;
+            if(output && output.options){
```

Comment:
other coding style again? ;-) Spaces missing again (sorry to bother with such stuff)